install_vendored_static_xcframework/before/Podfile

```diff
@@ -0,0 +1,7 @@
+
+target 'XCFrameworkIntegration' do
+  platform :ios, '13.0'
+  use_frameworks!
+
+  pod 'BananaLib', :path => 'BananaLib', :testspecs => ['Tests']
```

Comment:
mind pulling out `BananaLib` and `CoconutLib` from the existing vendored_xcframework test folder so we don't have to check it into the repo twice? will also make it easier to update those fixtures since there won't be a duplicate

basically moving this folder: https://github.com/CocoaPods/cocoapods-integration-specs/tree/master/install_vendored_xcframework/before/BananaLib

to a top-level `BananaLib` folder so it can be reused, then use `pod 'BananaLib', :path => '../../BananaLib'` here

Reply:
The `BananaLib` podspec is referencing a static `CoconutLib` so i'm not sure if we can do that.

Reply:
Ah I was thinking of something in the CocoaPods examples - my bad. this is fine 👍